internal/commands/user/delete/flags.go

```diff
@@ -0,0 +1,64 @@
+package delete
+
+import (
+	"fmt"
+	"strings"
+)
+
+const (
+	flagUsers      = "user"
+	flagUsersShort = "u"
+	flagUsersUsage = "users to be deleted"
+
+	flagInteractive      = "interactive-filter"
+	flagInteractiveShort = "i"
```

Comment:
I ran into conflicts when testing out this branch with a profile since the profile flag shorthand is also 'I'. Unfortunately 'f' seems to be taken as well, but maybe there is another letter that makes sense if we need to rename this.